README.md

```diff
@@ -26,17 +26,22 @@ File handler utilizing TUS and MinIO with DRS-Filer integration.
 - [Contact](#contact)
 
 ## Synopsis
-This application provides endpoints for uploading, downloading, and listing files in a MinIO bucket, with TUS protocol support for uploads, and CORS enabled for cross-origin requests.
+
+This application provides endpoints for uploading, downloading, and listing
+files in a MinIO bucket, with TUS protocol support for uploads, and CORS enabled
+for cross-origin requests.
 
 ## Basic Usage
 
 ## Installation
 
 ### Prerequisites
 
-This flask application requires a running instance of [minio](https://min.io/download)
+This flask application requires a running instance of
+[minio](https://min.io/download)
 
-Run the minio instance by executing the following command in the location where minio is installed
+Run the minio instance by executing the following command in the location where
+minio is installed
```

Comment:
Please use "MinIO" instead of "minio" and add a colon (`:`) at the end of the sentence.